ferrocene/doc/evaluation-report/src/rustfmt/tool-analysis.rst

```diff
@@ -55,12 +57,17 @@ Moreover, this analysis shows us that the likelihood of detecting these
 potential errors is very high. Therefore, the tool error detection class is
 **TD1**.
 
-Using clause 11.4.5.4 in part 8 of the [|iso_ref|] standard, we can conclude that in
-the worst case the Tool Classification Level is **TCL1**. Therefore we need no
-qualification methods.
+Using clause 11.4.5.4 in part 8 of the [|iso_ref|] standard, we can conclude
+that in the worst case the Tool Classification Level is **TCL1/ASIL D**.
```

Comment:
Why did you change the classifcation level to be TCL3/ASIL D? The TCL is just TCL3.

Reply:
Same reasoning as above.